src/components/shared/Entity/ExistingEntityCards/Cards/Existing.tsx

```diff
@@ -0,0 +1,175 @@
+import Editor from '@monaco-editor/react';
+import {
+    Box,
+    ButtonBase,
+    Collapse,
+    Paper,
+    Typography,
+    useTheme,
+} from '@mui/material';
+import {
+    CaptureQueryWithSpec,
+    MaterializationQueryWithSpec,
+} from 'api/liveSpecsExt';
+import { authenticatedRoutes } from 'app/routes';
+import ConnectorLogo from 'components/connectors/card/Logo';
+import CustomWidthTooltip from 'components/shared/CustomWidthTooltip';
+import {
+    alternateConnectorImageBackgroundSx,
+    monacoEditorComponentBackground,
+    semiTransparentBackground,
+    semiTransparentBackgroundIntensified,
+} from 'context/Theme';
+import { GlobalSearchParams } from 'hooks/searchParams/useGlobalSearchParams';
+import { isEmpty } from 'lodash';
+import LogRocket from 'logrocket';
+import { useState } from 'react';
+import { useIntl } from 'react-intl';
+import { useNavigate } from 'react-router';
+import { CustomEvents } from 'services/logrocket';
+import { stringifyJSON } from 'services/stringify';
+import { getPathWithParams } from 'utils/misc-utils';
+
+interface Props {
+    queryData: CaptureQueryWithSpec | MaterializationQueryWithSpec | null;
+}
+
+const EDITOR_HEIGHT = 396;
+
+const trackEvent = (
+    logEvent: CustomEvents,
+    data: CaptureQueryWithSpec | MaterializationQueryWithSpec
+) => {
+    LogRocket.track(logEvent, {
+        id: data.id,
+        connector_tag_id: data.connector_id,
+    });
+};
+
+function ExistingEntityCard({ queryData }: Props) {
```

Comment:
This is okay... but this probably should not have been architected like this. I think it would have been better to make a more common horizontal/long/whatever card and then built off that for ExistingEntity. But not big deal.

Reply:
Originally, the existing entity cards were not innately buttons and had an assortment of functions built into them (e.g., specification and endpoint config preview, options menu that surfaced the edit option and an advanced specification comparer, a bespoke _Edit_ CTA when the options menu was vetoed, etc.).

Given the existing entity cards share so much in common with the new entity card, I agree that a base should be defined.